docs/api/classifier/rocauc.rst

```diff
@@ -53,7 +53,7 @@ This leads to another metric, area under the curve (AUC), which is a computation
 
 
 Multi-class ROCAUC Curves
-#########################
+-------------------------
```

Comment:
Created the sub-section using `---` instead of `###` so that it renders properly in the table of contents.